src/monocle_apptrace/metamodel/spans/span_example.json

```diff
@@ -0,0 +1,140 @@
+{
+    "name": "llamaindex.retrieve",
+    "context": {
+        "trace_id": "0x93cd0bf865b3ffcc3cf9c075dc3e3797",
+        "span_id": "0x5d3f839e900bda24",
+        "trace_state": "[]"
+    },
+    "kind": "SpanKind.CLIENT",
+    "parent_id": "0x7a63d63e42ccac60",
+    "start_time": "2024-09-09T14:38:45.237182Z",
+    "end_time": "2024-09-09T14:38:45.620112Z",
+    "status": {
+        "status_code": "OK"
+    },
+    "attributes": {
+        "span.type": "Retrieval", 
+        "entity.count": 2,
+        "entity.1.name": "ChromaVectorStore",
+        "entity.1.type": "VectorDB.Chroma",
+        "entity.1.embedding-model-name": "BAAI/bge-small-en-v1.5",
+        "entity.2.name": "BAAI/bge-small-en-v1.5",
+        "entity.2.type": "Model.embedding",
+        "entity.2.model_name": "BAAI/bge-small-en-v1.5"
+    },
+    "events": [
+        {
+            "name": "data.input",
+            "timestamp": "timestamp",
+            "attributes": {
+                "context_input": "question: What is an americano?"
+            }
+        },
+        {
+            "name": "data.output",
+            "timestamp": "timestamp",
+            "attributes": {
+                "context_output": "Coffee is a hot drink made from the roasted and ground seeds (coffee beans) of a tropical shrub\nA latte consists of one or more shots of espresso, served in a glass (or sometimes a cup), into which hot steamed milk is added\nAmericano is a type of coffee drink prepared by diluting an espresso shot with hot water at a 1:3 to 1:4 ratio, resulting in a drink that retains the complex flavors of espresso, but in a lighter way"
+            }
+        }
+
+    ],
+    "links": [],
+    "resource": {
+        "attributes": {
+            "service.name": "coffee-bot"
+        },
+        "schema_url": ""
+    }
+},
+{
+    "name": "llamaindex.openai",
+    "context": {
+        "trace_id": "0x93cd0bf865b3ffcc3cf9c075dc3e3797",
+        "span_id": "0x8b6363e1937a4d7b",
+        "trace_state": "[]"
+    },
+    "kind": "SpanKind.CLIENT",
+    "parent_id": "0x7a63d63e42ccac60",
+    "start_time": "2024-09-09T14:38:45.622174Z",
+    "end_time": "2024-09-09T14:38:46.514120Z",
+    "status": {
+        "status_code": "OK"
+    },
+    "attributes": {
+        "span.type": "Inference",
+        "entity.count": 2,
+        "entity.1.name": "AzureOpenAI",
+        "entity.1.type": "Inference.Azure_oai",
+        "entity.1.provider_name": "openai.azure.com",
+        "entity.1.deployment": "kshitiz-gpt",
+        "entity.1.inference_endpoint": "https://okahu-openai-dev.openai.azure.com/",
+
+        "entity.2.name": "gpt-35-turbo",
+        "entity.2.type": "Model.LLM",
+        "entity.2.model_name": "gpt-35-turbo",
+        "entity.2.temperature": 0.1
```

Comment:
temperature is per request and should be part of `metadata` in events

Reply:
Ah ok. will fix it.